src/features/DisplayControls/NursingTasks/styles/AdministeredMedicationList.scss

```diff
@@ -0,0 +1,27 @@
+.bx--structured-list {
+  margin: 0;
+}
+
+.time, .status {
+  position: relative;
+  padding-left: 8px;
+}
+
+.time {
+  top: -3px;
+}
+
+.status {
+  top: -4px;
+}
+
+.drug-name-cell {
+  width: 55%;
+}
+
+.time-cell {
```

Comment:
@tanyaa-tw We already have a time-cell class in Drug chart, With this change the border is breaking in drug chart. Maybe change the class name.